src/internal/types.ts

```diff
@@ -67,6 +67,7 @@ export type NetInfoWifiState = NetInfoConnectedState<
   NetInfoStateType.wifi,
   {
     ssid: string | null;
+    bssid: string | null,
```

Comment:
```suggestion
    bssid: string | null;
```

Reply:
Very well spotted. Somehow my linter didn't warn me on that. Fixed :)